hyperseti/normalize.py

```diff
@@ -66,18 +66,18 @@ def normalize(data,  mask=None, poly_fit=0):
         d_std_ifs[ii]  = dstd
 
     t1p = time.time()
-    logger.info(f"Poly fit time: {(t1p-t0p)*1e3:2.2f}ms")
+    ### logger.info(f"Poly fit time: {(t1p-t0p)*1e3:2.2f}ms")
 
     flag_fraction =  N_flagged / N_tot
     flag_correction =  N_tot / (N_tot - N_flagged) 
-    logger.info(f"Flagged fraction: {flag_fraction:2.4f}")
-    if flag_fraction > 0.2:
-        logger.warning(f"High flagged fraction: {flag_fraction:2.3f}")
+    ### logger.info(f"Flagged fraction: {flag_fraction:2.4f}")
```

Comment:
IIRC these were commented out due to a runtime issue with older environment -- can these be added back in? The high flagged fraction warning is important

Reply:
Agreed.